package.json

```diff
@@ -56,13 +58,11 @@
     "tslint": "5.20.1",
     "typescript": "3.9.3",
     "apollo-server-express": "2.14.1",
-    "graphql-tools": "6.0.5",
     "keycloak-connect": "10.0.1"
   },
   "peerDependencies": {
     "graphql": "^0.12.0 || ^0.13.0 || ^14.0.0 || ^15.0.0",
-    "graphql-tools": ">=4.0.0",
-    "keycloak-connect": ">=7.0.0"
+    "keycloak-connect": ">=9.0.0"
```

Comment:
Keycloak is now at version 10. We need to migrate to version 9 as 8,7,6 stopped working due to extensive typescript changes